pom.xml

```diff
@@ -170,6 +172,12 @@
                     <version>3.10.1</version>
                     <configuration>
                         <parameters>true</parameters>
+                        <source>11</source>
+                        <target>11</target>
+<!--                        <release>11</release>-->
```

Comment:
What's the end goal here? Should we not use 17 for testing? And why the change to source/target? I think I've asked about this before, but can't seem to find it. Sorry for asking again.